docs/content/en/docs/tutorials/add-tasks.md

```diff
@@ -15,9 +15,206 @@ When Keptn is successfully monitoring your deployments, it can also run arbitrar
 - pre-deployment (before the pod is scheduled) and
 - post-deployment (after the post is scheduled)
 
-Go to the [pre and post deployment checks page](../implementing/integrate#pre--and-post-deployment-checks) to find out more.
+> Pre and post deployments can also run on a KeptnApp level.
+> See [annotations to KeptnApp](../implementing/integrate/_index.md#annotations-to-keptnapp)
 
-_!
-TODO: Move the content from the above page to here.
-Help us improve the Keptn documentation by
-[opening an issue](https://github.com/keptn/lifecycle-toolkit/issues/new) to fix this documentation !_
+## Prerequisites: Deploy webhook sink
+
+During this exercise, you will configure Keptn to trigger a webhook before and after a deployment has successfully completed.
+
+For demo purposes, a place is required to send those request.
+Install the [open source webhook.site tool](https://github.com/webhooksite/webhook.site/tree/master/kubernetes) now.
+
+This will provide a place, on cluster, to send (and view) web requests.
+
+> Note: If you have your own endpoint, you can skip this step.
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/webhooksite/webhook.site/master/kubernetes/namespace.yml
+kubectl apply -f https://raw.githubusercontent.com/webhooksite/webhook.site/master/kubernetes/redis.deployment.yml
+kubectl apply -f https://raw.githubusercontent.com/webhooksite/webhook.site/master/kubernetes/laravel-echo-server.deployment.yml
+kubectl apply -f https://raw.githubusercontent.com/webhooksite/webhook.site/master/kubernetes/webhook.deployment.yml
+kubectl apply -f https://raw.githubusercontent.com/webhooksite/webhook.site/master/kubernetes/service.yml
+```
+
+Wait until all pods are running in the `webhook` namespace then port-forward and view the webhook sink page:
+
+```shell
+kubectl -n webhook wait --for=condition=Ready pods --all
+kubectl -n webhook port-forward svc/webhook 8084
+```
+
+Open a browser and go to `http://localhost:8084`
+
+You should see a page like this with a unique URL (your ID will be different).
+
+![webhook.site page](assets/webhook.site.1.png)
```

Comment:
this is how we embed images from the same root folder .. i.e. https://github.com/keptn/lifecycle-toolkit/blob/main/docs/content/en/docs/implementing/dora/_index.md not sure why html test fails hopefully it will fix itself after merge 😭